src/kerchunk/gran_to_cat.py

```diff
@@ -0,0 +1,197 @@
+import os
+import warnings
+import fsspec
+import ujson
```

Comment:
i have had issues with parsing attirbutes from kerchunk references using `orjson`. Since we really don't need the extra speed anyways for this particular use-case I'll switch to using the stdlib `json` module.